packages/test/src/workflows/signal-start-otel.ts

```diff
@@ -0,0 +1,29 @@
+import * as workflow from '@temporalio/workflow';
+import {
+  OpenTelemetryInboundInterceptor,
+  OpenTelemetryOutboundInterceptor,
+  OpenTelemetryInternalsInterceptor,
+} from '@temporalio/interceptors-opentelemetry/lib/workflow';
+
+export const startSignal = workflow.defineSignal('startSignal');
+
+const { a, b, c } = workflow.proxyLocalActivities({
+  scheduleToCloseTimeout: '1m',
+});
```

Comment:
where are these activities coming from? I don't see any activities registered on the test workers, maybe i'm missing something though

Reply:
I did remove the code to create the histories since the tests only replay history to check for verify there isn't NDE. You can view the code used at https://github.com/temporalio/sdk-typescript/pull/1803/commits/abf1134f3ba0b2ba1828e42395af90c83e6cadb5